boxsdk/session/session.py

```diff
@@ -285,11 +286,14 @@ def _get_retry_after_time(self, attempt_number, retry_after_header):
         :return:                        Number of seconds to wait before retrying.
         :rtype:                         `Number`
         """
-        min_randomization = 1 - self._retry_randomization_factor
-        max_randomization = 1 + self._retry_randomization_factor
-        randomization = (random.uniform(0, 1) * (max_randomization - min_randomization)) + min_randomization
-        exponential = math.pow(2, attempt_number)
-        return exponential * self._retry_base_interval * randomization
+        try:
```

Comment:
Is there a way to check for the presence of `retry_after_header` before doing the `try` statement?  I don't know about Python, but there is usually an overhead associated with relying on the exception to be thrown instead of a condition statement.  And I don't know how often we expect `retry_after_header` to be missing ... is it usually present?

Reply:
We can add an if statement that checks whether the header is there.

Reply:
I'm new to Python, I could be wrong:
https://stackoverflow.com/questions/1835756/using-try-vs-if-in-python

Reply:
Added an if check. Doesn't make sense to cast if we don't need to.